proto/net.proto

```diff
@@ -106,6 +108,53 @@ message Weights {
     optional Layer ln2_betas = 6;
   }
 
+  message PolicyHead {
+    // Policy head
+    // Extra convolution for AZ-style policy head
+    optional ConvBlock policy1 = 100;
+    optional ConvBlock policy = 101;
+    // Add policy encoders for legacy support
+    repeated EncoderLayer pol_encoder = 102;
+    optional uint32 pol_headcount = 103;
+    optional Layer ip_pol_w = 1;
+    optional Layer ip_pol_b = 2;
```

Comment:
Those two are not used as they are shared among all heads using the values given in `PolicyHeads`. Do we need to keep them for some reason?

Reply:
If my assumption here is wrong and we need to keep them, then the corresponding fields in `PolicyHeads` should be renamed.

Reply:
These ones are associated with older nets and should probably remain. We can rename the ones in `PolicyHeads`.

Reply:
This has been kept, as per the latest approach we're using.